pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Related Issue
+- Link for the issue
+- Description of the issue
+
+# Proposed Changes
+- Change 1
+- Change 2
+- Change 3
+
+# Additional Info
+- Additional content or any information
+
+# Checklist
+- [ ] Have you tested your code?
+- [ ] Does it follow style guides?
+- [ ] Does it also include translations?
```

Comment:
We have not created the web application itself yet, so I think that the translations can be done at a later stage